Dockerfile

```diff
@@ -30,6 +30,7 @@ COPY root/ /
 RUN set -x && \
     apt update && \
     apt install -y \
+        file \
```

Comment:
Why is this necessary?

Reply:
I'm using it in my post execution script so I can find directories which don't contain video files (so only left over metadata from yt-dlp). 

Happy to remove if needed but given it's small package seemed a nice to have with the introduction of post processing scripts that might want to do similar things :-)

Reply:
Okey, let's keep it.